### PR TITLE
add email todo support

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

File: shared/constants/people.js [hunk 1]
```diff
@@ -90,6 +94,7 @@ export const todoTypeToIcon: {[key: Types.TodoType]: IconType} = {
   proof: isMobile ? 'icon-onboarding-proofs-48' : 'icon-onboarding-proofs-32',
   team: isMobile ? 'icon-onboarding-team-48' : 'icon-onboarding-team-32',
   teamShowcase: isMobile ? 'icon-onboarding-team-publicity-48' : 'icon-onboarding-team-publicity-32',
+  addEmail: 'icon-fancy-email-sent-144-x-48.png',
```
Comment (heronhaye) [inline shared/constants/people.js]: placeholder

**Comment by heronhaye** [comment]
I believe this is not passing since it isn't supported on webserver yet?

**Comment by zapu** [comment]
I rebase on master to grab rest of noemail fixed we merged and fixed bunch of linter errors.

Needs an icon from design.

**Comment by heronhaye** [comment]
TODO: Make todo connector pop open the "Add email" modal.

**Comment by buoyad** [comment]
@heronhaye want to close this now that https://github.com/keybase/client/pull/18246 is in?

**Pull request closed without merging**